### PR TITLE
Admin screen with Users tab populated

### PR DESCRIPTION
**Pull request by sadhnaahuja**

### CODE REVIEW

File: family2family/src/main/resources/data-dev.sql [hunk 1]
```diff
@@ -264,13 +264,10 @@ INSERT INTO DIAGNOSIS(DIAGNOSIS_ID, DESCRIPTION) VALUES
 
 
 --Dummy Data
-insert into users(id, admin, active) values('work@pls', false, false);
```
Comment (theismanj) [inline family2family/src/main/resources/data-dev.sql:267]: Can you change this to not remove my team's email addresses?

File: family2family/src/main/resources/static/js/admin.jsx [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+                </a>
+               <div className="edit-collapse">
+                   <EditUserRow id={this.state.id} firstName={this.state.firstName}
+                    lastName={this.state.lastName} admin={this.state.admin} active={this.state.active}/>
+               </div>
+            </div>
+        )
+    }
+}
+class UserTable extends React.Component {
```
Comment (theismanj) [inline family2family/src/main/resources/static/js/admin.jsx]: Nitpicking, but can we have better formatting here? Add a line break before the new class and indent the constructor.

File: family2family/src/main/resources/static/js/admin.jsx [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+                <TabContentFrames />
+            </div>
+        )
+    }
+}
+
+class EditUserRow extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {id:props.id, firstName:props.firstName, lastName:props.lastName, admin:props.admin, active:props.active};
```
Comment (theismanj) [inline family2family/src/main/resources/static/js/admin.jsx]: What is the purpose of setting the state with all the properties here when we don't use the state we use props below?

File: family2family/src/main/resources/static/js/admin.jsx [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+    }
+
+    handleChange(event) {
+        this.setState({value: event.target.value});
```
Comment (theismanj) [inline family2family/src/main/resources/static/js/admin.jsx]: When I hit this change event I get the following error 
`Uncaught TypeError: Cannot read property 'setState' of undefined`

**Review by theismanj — DISMISSED** [review]

File: family2family/src/main/resources/static/js/admin.jsx (hunk 4 above)
Comment (sadhnaahuja) [reply family2family/src/main/resources/static/js/admin.jsx]: I created https://github.com/coop-code-challenge/cbtf/issues/105 to fix this one later.

**Review by toscott — APPROVED** [review]
Pulled down and ran it. Looks like the users are coming back and displaying on the screen. Screen could still use some love, but feels like its a decent starting place 👍

**Pull request merged**